src/Servers/Kestrel/Core/src/Internal/Http/ChunkWriter.cs

```diff
@@ -28,7 +29,7 @@ public static int BeginChunkBytes(int dataCount, Span<byte> span)
             count = (total >> 2) + 3;
 
             var offset = 0;
-            ref var startHex = ref _hex[0];
+            ref var startHex = ref MemoryMarshal.GetReference(Hex);
```

Comment:
This seems a little unnatural though.

Reply:
As it's a ROS there is no other (safe) way to get a reference from it. The C#-compiler won't allow it.
`Cannot use property 'ReadOnlySpan<byte>.this[int]' as a ref or out value because it is a readonly variable`.